testdata/p4_16_dpdk_errors_outputs/pna-add-on-miss-err1.p4.bfrt.json

```diff
@@ -0,0 +1,167 @@
+{
+  "schema_version" : "1.0.0",
```

Comment:
Do you want to require checking in expected .bfrt.json files for programs that you expect an error message for?  Do you expect to get much test coverage out of such expected output files?

Reply:
I think we can avoid this check. I will update